### ISSUE TITLE
Use Language Embedding to Show Most Similar Notes

### ISSUE THREAD

**Issue by zachdoty** [louis030195/obsidian-ava#40]
A sidebar the shows the most similar notes would be awesome!

**Comment by anubop** [comment]
+1 if we could embed our notes and search in embedding space, it would be helpful for resurfacing note, and automatically drawling links.

**Comment by zachdoty** [comment]
Maybe even tie it in to https://www.rewind.ai/ - and make it so each obsidian page you're looking at auto-surfaces the most similar content you've seen or consumed in the past weeks - semantic search for your life, surfaced in-context when you're looking at any note.

**Comment by hotkartoffel** [comment]
that would be really cool, we're focusing on the write/rewrite/ai image this week and will be spending the next week improving the whole semantic search experience.

Just curious:
@zachdoty @anubop would you be comfortable having your notes sent to the cloud? (having a local api causes a lot of UX issues so we're exploring every option)

(love rewind ai that's a great idea, will def look into it :) )

**Comment by arminta7** [comment]
I'm extremely interested in this as well. I'm comfortable having mine sent to the cloud.

**Comment by zachdoty** [comment]
@hotkartoffel , would you and/or @louis030195 be available to jump on a GMeet video call to discuss some ideas and brainstorm technical solutions? https://calendly.com/zachdoty

Also want to video share with you so I can show you what I'm building for Obsidian... which also uses LLMs and dove-tails nicely with what you're doing...

**Comment by hotkartoffel** [comment]
Hi @zachdoty just looking at this rn

I checked your calendly, but it seems we're on quite diff timezones.

Would be happy to have a call though. Most likely next week (after new year) if that still works with it you :)

**Comment by hotkartoffel** [comment]
> I'm extremely interested in this as well. I'm comfortable having mine sent to the cloud.

We just released a stable version of our embeddings.

**Comment by zachdoty** [comment]
> Would be happy to have a call though. Most likely next week (after new year) if that still works with it you :)

Yeah @hotkartoffel , I'm free for a call anytime. Can be flexible outside of those hours too. Wanna connect on telegram or discord? https://zachdoty.t.me
